agent/util-scripts/test-bin/common-tm-cleanup

```diff
@@ -10,12 +10,26 @@ function _kill_pid {
     fi
 }
 
+function _destroy_redis {
+    redis_pids=$(ps uxww | grep -E "[r]edis-server" | awk '{print $2}')
```

Comment:
OK, since I apparently made this comment in a stacked PR without realizing it... 😄 

What does the `[r]` represent here? A mandatory set with a single element should be exactly the same as the single character, right? Why isn't this just `-F redis-server`?

Reply:
It prevents the `grep` from matching itself.